__tests__/app.test.js

```diff
@@ -75,10 +74,38 @@ describe("/api/articles/:article_id", () => {
   });
   test("GET 400: responds with correct status and error message when requesting an invalid ID", () => {
     return request(app)
-    .get('/api/articles/forklift')
-    .expect(400)
-    .then(({body: {msg}}) => {
-        expect(msg).toBe('Bad request');
-    })
+      .get("/api/articles/forklift")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request");
+      });
+  });
+});
+describe("/api/articles", () => {
+  test("GET 200: responds with an array of all articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(13);
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+        });
```

Comment:
Great that we've ensured the properties exist - but we could make this an even stronger test by also checking the type of the value that is returned.
There's a few ways we can do this - toHaveProperty can actually accept a second argument for the expected value like so:
          expect(article).toHaveProperty("author", expect.any(String));
          
OR, you can use the jest matcher .toMatchObject for a similar purpose:

expect(article).toMatchObject({
author : expect.any(String)
<... other properties>
})